users/models.py

```diff
@@ -316,4 +316,4 @@ def __str__(self):
 class BlockList(TimestampedModel):
     """ A user's blocklist model """
 
-    hashed_email = models.CharField(max_length=128, blank=True, default="")
+    hashed_email = models.CharField(max_length=128, default="")
```

Comment:
I think you also need to remove the default, since it'll fail now.

Also I think you need to update your PR description because this only fixes #2229

Reply:
This PR only fixes #2225